tempto-core/src/main/java/com/teradata/tempto/fulfillment/table/hive/tpcds/TpcdsDataSource.java

```diff
@@ -92,6 +95,13 @@ public String revisionMarker()
     }
 
     @Override
+    public Optional<TableStatistics> getStatistics()
+    {
+        TableStatisticsRepository tableStatisticsRepository = new TableStatisticsRepository();
+        return Optional.of(tableStatisticsRepository.load("tpcds", scaleFactor, table.name()));
```

Comment:
This will fail for scale factor for which we do not have stats json file.
`load` should probably return `Optional`.